docker-image/v0.12/alpine-logzio/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# AUTOMATICALLY GENERATED
+# DO NOT EDIT THIS FILE DIRECTLY, USE /templates/Dockerfile.erb
+
+FROM fluent/fluentd:v0.12.33
+MAINTAINER Eduardo Silva <eduardo@treasure-data.com>
+USER root
+WORKDIR /home/fluent
+ENV PATH /home/fluent/.gem/ruby/2.3.0/bin:$PATH
+
+RUN set -ex \
+    && apk update && apk upgrade \
+    && apk add --no-cache --virtual .build-deps \
+        build-base \
+        ruby-dev \
+        libffi-dev \
+    && echo 'gem: --no-document' >> /etc/gemrc \
+    && gem install fluent-plugin-secure-forward \
+    && gem install fluent-plugin-record-reformer \
+    && gem install fluent-plugin-logzio \
```

Comment:
fluent-plugin-logzio v0.0.15 or newer is not compatible for Fluentd v0.12.
Could you specify fluent-plugin-logzio version explicitly?

```
&& gem install fluent-plugin-logzio -v 0.0.14 \
```

ref: https://github.com/logzio/fluent-plugin-logzio/tree/db20fd0887f15ebfb1ba5b241fb308d0fc2454d7#requirements